ansible_runner/runner.py

```diff
@@ -186,6 +229,10 @@ def run(self):
                 f.write(str(data))
         if self.config.directory_isolation_path and self.config.directory_isolation_cleanup:
             shutil.rmtree(self.config.directory_isolation_path)
+        if self.config.resource_profiling:
+            cmd = ['cgdelete', '-g', 'cpuacct,memory,pids:ansible-runner-{}'.format(self.config.ident)]
```

Comment:
Do you want to use `resource_profiling_base_cgroup`, not hard-coded `ansible-runner-`?

Reply:
Instead of making this string again, could you just reuse `cgroup_path` from above?